### PR TITLE
Add additional view and describe permissions

### PR DESCRIPTION
**Pull request by petecheslock**

### CODE REVIEW

**Comment by jcderose** [comment]
As long as you have `elasticmapreduce:List*` access already, this looks good to me.

File: aws-cli/resource-discovery-policy.json [hunk 1]
```diff
@@ -54,6 +55,8 @@
                 "s3:GetAnalyticsConfiguration",
                 "s3:GetBucket*",
                 "s3:GetReplication*",
+                "s3:GetStorageLensDashboard",
+                "s3:ListStorageLensConfigurations",
```
Comment (jcderose) [inline aws-cli/resource-discovery-policy.json:59]:
```suggestion
                "s3:ListStorageLensConfiguration",
```

File: cloudformation/duckbill-iam-role.yml [hunk 2]
```diff
@@ -123,6 +124,8 @@ Resources:
               - 's3:GetAnalyticsConfiguration'
               - 's3:GetBucket*'
               - 's3:GetReplication*'
+              - 's3:GetStorageLensDashboard'
+              - 's3:ListStorageLensConfigurations'
```
Comment (jcderose) [inline cloudformation/duckbill-iam-role.yml:128]:
```suggestion
              - 's3:ListStorageLensConfiguration'
```

File: terraform/duckbill-iam-role.tf [hunk 3]
```diff
@@ -136,6 +137,8 @@ data "aws_iam_policy_document" "DuckbillGroupResourceDiscovery_policy_document"
       "s3:GetAnalyticsConfiguration",
       "s3:GetBucket*",
       "s3:GetReplication*",
+      "s3:GetStorageLensDashboard",
+      "s3:ListStorageLensConfigurations",
```
Comment (jcderose) [inline terraform/duckbill-iam-role.tf:141]:
```suggestion
      "s3:ListStorageLensConfiguration",
```

**Review by jcderose — CHANGES_REQUESTED** [review]
Looks like the `Configuration` action is singular, not plural.

Also, I don't see `s3:ListStorageLensConfigurations` in the ViewOnlyAccess policy so you may need to add that as well.

**Comment by petecheslock** [comment]
Looking at the role it looks like the "list" permission is plural.  
Screen Shot 2020-11-25 at 9 46 14 AM

**Comment by jcderose** [comment]
Blarg, I was looking at the wrong thing. I thought I was making a change to `s3:GetStorageLensConfiguration`, not `s3:ListStorageLensConfigurations`. 🤦

**Review by jcderose — APPROVED** [review]
Blarg, I was looking at the wrong thing. I thought I was making a change to `s3:GetStorageLensConfiguration`, not `s3:ListStorageLensConfigurations`. 🤦

**Pull request merged**